packages/aws-rfdk/lib/deadline/lib/thinkbox-docker-images.ts

```diff
@@ -223,9 +231,10 @@ AWS Thinkbox EULA.
       validate(): string[] {
         const errors: string[] = [];
 
-        // Users must accept the AWS Thinkbox EULA to use the container images
-        if (thisConstruct.userAwsThinkboxEulaAcceptance !== AwsThinkboxEulaAcceptance.USER_ACCEPTS_AWS_THINKBOX_EULA) {
-          errors.push(ThinkboxDockerImages.AWS_THINKBOX_EULA_MESSAGE);
+        // Users must accept the AWS Customer Agreement and IP License to use the container images
```

Comment:
```suggestion
        // Users must accept the AWS Customer Agreement and AWS IP License to use the container images
```
also IP not obvious as for me, so maybe
```suggestion
        // Users must accept the AWS Customer Agreement and AWS Intellectual Property License to use the container images
```

Reply:
Yup. Good point. Will fix.